target/linux/ramips/dts/mt7621_ampedwireless_ally-00x19k.dts

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_ampedwireless_ally.dtsi"
+
+/ {
+	compatible = "ampedwireless,ally-00x19k", "mediatek,mt7621-soc";
```

Comment:
I wonder whether we should just use "amped" for the vendor part here ...

Reply:
I thought about that but ultimately decided this was unambiguous.

Reply:
@adschm FWIW, elsewhere in ramips we have vendors "alfa-network", "alphanetworks", "mofinetwork", "rakwireless" and "arcwireless".